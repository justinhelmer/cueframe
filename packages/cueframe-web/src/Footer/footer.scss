#footer {
  border-top: 2px solid var(--light)
}