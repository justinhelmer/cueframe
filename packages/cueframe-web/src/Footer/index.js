import React from 'react';
import './footer.scss';

export default () => (
  <div id="footer" className="text-center py-5">Copyright 2019</div>
);
