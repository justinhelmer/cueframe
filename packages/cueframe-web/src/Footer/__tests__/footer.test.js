import React from 'react';
import Footer from '..';
import renderer from 'react-test-renderer';

describe('<Footer/>', () => {
  describe('Snapshot', () => {
    it('renders correctly', () => {
      const tree = renderer
        .create(<Footer/>)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
