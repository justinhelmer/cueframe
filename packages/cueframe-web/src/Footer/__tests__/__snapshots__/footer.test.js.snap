// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> Snapshot renders correctly 1`] = `
<div
  className="text-center py-5"
  id="footer"
>
  Copyright 2019
</div>
`;
